src/Approximations/decompositions.jl

```diff
@@ -442,3 +443,30 @@ The template direction approximation of the projection of `S`.
     M = sparse(1:length(block), block, ones(N, length(block)), length(block), n)
     return overapproximate(M * S, directions)
 end
+
+"""
+    project(S::LazySet{N},
+            block::AbstractVector{Int},
+            n::Int
+           )::LinearMap where {N<:Real}
+
+Project a high-dimensional set to a low-dimensional set.
```

Comment:
Maybe while you are at it, add a more precise description of the method (but I am not sure if this is better or worse):

    Project a high-dimensional set to a low-dimensional set by a lazy linear map.